### PR TITLE
XAMARIN-43421 Prepared UG content for EffectsView Color opacity property

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
Hi @vigneshrameshsync,
              I have prepared UG content for EffectsView colour opacity customization properties.

### CODE REVIEW

File: Xamarin/Effects-View/Customization.md [hunk 1]
```diff
@@ -302,6 +302,94 @@ var effectsView = new SfEffectsView
...
+    RippleColorOpacity = 0.5
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## SelectionColorOpacity
+
+The [`SelectionColorOpacity`]() property of [`SfEffectsView`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.EffectsView.SfEffectsView.html) is used to gets or sets a value that determines the opacity of the selection effect.
```
Comment (vigneshrameshsync) [inline Xamarin/Effects-View/Customization.md:365]: Please change the content by referring to the below UG 
https://help.syncfusion.com/winui/radial-gauge/axes#radius-customization

File: Xamarin/Effects-View/Customization.md [hunk 2]
```diff
@@ -302,6 +302,94 @@ var effectsView = new SfEffectsView
...
+{
+    LongPressEffects = SfEffects.Selection,
+    SelectionColorOpacity = 0.5
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Selection Color Opacity customization](Customization_images/EffectsView_Selection_Effect_Opacity.png)
```
Comment (vigneshrameshsync) [inline Xamarin/Effects-View/Customization.md:391]: It is just alt text. So, please avoid unwanted cast changes.

File: Xamarin/Effects-View/Customization.md [hunk 3]
```diff
@@ -302,6 +302,94 @@ var effectsView = new SfEffectsView
...
+
+The [`HighlightColorOpacity`]() property of [`SfEffectsView`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.EffectsView.SfEffectsView.html) is used to gets or sets a value that determines the opacity of the highlight effect.
+
+{% tabs %} 
+
+{% highlight xaml %} 
+
+<syncEffectsView:SfEffectsView
+    TouchDownEffects="Highlight"
+    HighlightColorOpacity="0.5">
```
Comment (vigneshrameshsync) [inline Xamarin/Effects-View/Customization.md:315]: Whether the images captured with default color with opacity values? or customized at sample level?

**Review by vigneshrameshsync — CHANGES_REQUESTED** [review]

File: Xamarin/Effects-View/Customization.md [hunk 4]
```diff
@@ -302,6 +302,94 @@ var effectsView = new SfEffectsView
 
 ![Rotation Angle customization](Customization_images/EffectsView_Rotation.png)
 
+## HighlightColorOpacity
+
+The [`HighlightColorOpacity`]() property of [`SfEffectsView`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.EffectsView.SfEffectsView.html) is used to gets or sets a value that determines the opacity of the highlight effect.
```
[the remark this answers is not in the document]
Comment (TamilarasanGunasekaran) [reply Xamarin/Effects-View/Customization.md]: I have made changes in description.

File: Xamarin/Effects-View/Customization.md [hunk 5]
```diff
@@ -302,6 +302,94 @@ var effectsView = new SfEffectsView
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Highlight Color Opacity customization](Customization_images/EffectsView_HighlightEffect_Opacity.png)
+
+## RippleColorOpacity
+
+The [`RippleColorOpacity`]() property of [`SfEffectsView`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.EffectsView.SfEffectsView.html) is used to gets or sets a value that determines the opacity of the ripple effect.
```
[the remark this answers is not in the document]
Comment (TamilarasanGunasekaran) [reply Xamarin/Effects-View/Customization.md]: I have made changes in description.

File: Xamarin/Effects-View/Customization.md (hunk 1 above)
Comment (TamilarasanGunasekaran) [reply Xamarin/Effects-View/Customization.md]: I have made changes in description.

File: Xamarin/Effects-View/Customization.md (hunk 2 above)
Comment (TamilarasanGunasekaran) [reply Xamarin/Effects-View/Customization.md]: I have changed.

File: Xamarin/Effects-View/Customization.md (hunk 3 above)
Comment (TamilarasanGunasekaran) [reply Xamarin/Effects-View/Customization.md:315]: Its sample level image.

**Comment by TamilarasanGunasekaran** [comment]
rebuild

**Comment by TamilarasanGunasekaran** [comment]
rebuild

**Review by vigneshrameshsync — APPROVED** [review]
The changes are fine.

**Pull request merged**